pages/dkp/kommander/2.2/release-notes/index.md

```diff
@@ -2,10 +2,185 @@
+
+### New features and capabilities
+
+#### DKP major version upgrade
+
+Version 2.2 represents a major upgrade that moves forward D2IQ's architecture to give you access to the next generation of centralized Kubernetes and smart cloud native applications. It incorporates ClusterAPI as a major re-architecture in its management of production Kubernetes clusters. [ClusterAPI](https://cluster-api.sigs.k8s.io/introduction.html), or CAPI, enables declarative creation, configuration, and management of clusters. Declarative mode is a Kubernetes best practice that simplifies lifecycle tasks, making them more consistent and more repeatable. 2.2 enhances your existing clusters to use a new architecture.
+
+For more information on planning your upgrade, see the [DKP Major Version upgrade](/dkp/konvoy/2.2/major-version-upgrade/) guide.
+
+#### Provision Kubernetes Infrastructure from DKP
+
+This version allows for provisioning and managing of Kubernetes clusters, making it easier than ever to get your infrastructure up and running quickly.
+
+For more information on provisioning, see [Managing Clusters](../../clusters/).
+
+#### Continuous deployment
+
+Version 2.2 supports continuous delivery/deployment using Flux, which is designed for Kubernetes and supports multi-cluster and multi-tenant use cases. Configure Projects with GitOps-based Continuous Deployments using FluxCD, which enables canary and A/B deployments, as well as roll-back. DKP now uses a completely declarative approach, where what you define for production is what you get, without the need to monitor and manually intervene when something goes wrong.
+
+For more information on setting up continuous deployment using Flux, see [Continuous deployment](../../projects/project-deployments/continuous-delivery).
+
+#### Licensing through Amazon Marketplace
+
+You can purchase a license for DKP through the AWS Marketplace. In the UI, you can see information such as the license status (valid or expired), the license capacity (number of cores or clusters), and expiration date.
+
+### Component updates
+
+The following services and service components have been upgraded to the listed version:
+
+- centralized-grafana: 17.2.1
+- centralized-kubecost: 0.20.0
+- cert-manager: 0.2.7
+- dex: 2.20.5
+- external-dns: 2.20.5
+- fluent-bit: 0.16.2
+- gatekeeper: 0.6.8
+- grafana-logging: 6.13.9
+- grafana-loki: 0.33.1
+- istio: 1.9.1
+- jaeger: 2.21.0
+- karma: 2.0.0
+- kiali: 1.29.1
+- kube-oidc-proxy: 0.2.5
+- kube-prometheus-stack: 17.2.1
+- kubecost: 0.20.0
+- kubefed: 0.8.1
+- kubernetes-dashboard: 5.0.2
+- kubetunnel: 0.0.7
+- logging-operator: 3.13.0
+- metallb: 0.12.2
+- minio-operator: 4.1.7
+- nvidia: 0.4.3
+- project-grafana-logging: 6.13.9
+- project-grafana-loki: 0.33.1
+- project-logging: 1.0.0
+- prometheus-adapter: 2.11.1
+- reloader: 0.0.99
+- thanos: 0.4.5
+- traefik: 10.3.0
+- traefik-forward-auth: 0.3.2
+- velero: 3.1.3
+
+## Known issues
```

Comment:
leave known issues as a heading, but remove all the things below. let's hope we don't have any known issues in 2.2.